### DISCUSSION TITLE
Flows: Use functions in transform payload

### DISCUSSION THREAD

**Discussion by sebastiansmithomm** [directus/directus#13889] [Archive]
Hello, I've just discovered flows - this is an awesome feature! I am wondering if there is a way to do basic functions such as compute an average of an array variable or multiply an integer using the 'transform payload' operation? If not, are there any alternatives apart from creating my own webhook pointing to a cloud function. 

Thanks!

**Comment by rijkvanzanten** [comment]
There's currently no way to run basic logic against the payload values. We're currently wrapping up support for custom operations as an extension type, at which point you could use any javascript to modify the values in any way you'd like 🙂
